### PR TITLE
WRN-5318: Added units tests for ContextualMenuDecorator

### PR DESCRIPTION
**Pull request by daniel-stoian-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Added unit tests for ContextualMenuDecorator

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRN-5318

### Comments

Enact-DCO-1.0-Signed-off-by: Daniel Stoian daniel.stoian@lgepartner.com

### CODE REVIEW

File: ContextualMenuDecorator/tests/ContextualMenuDecorator-specs.js [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+import {ContextualMenuDecorator} from '../ContextualMenuDecorator';
+import Button from '../../Button';
+
+const ContextualButton = ContextualMenuDecorator(Button);
+const items = new Array(3).fill().map((i, index) => `Option ${index + 1}`);
+
+describe('ContextualMenuDecorator Specs', () => {
+	test('should render component into FloatingLayer if open', () => {
+		const Root = FloatingLayerDecorator('div');
+
```
Comment (alexandrumorariu) [inline ContextualMenuDecorator/tests/ContextualMenuDecorator-specs.js:15]: Remove this empty line(same below)

**Review by alexandrumorariu — APPROVED** [review]
LGTM

File: ContextualMenuDecorator/tests/ContextualMenuDecorator-specs.js [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+import {ContextualMenuDecorator} from '../ContextualMenuDecorator';
+import Button from '../../Button';
+
+const ContextualButton = ContextualMenuDecorator(Button);
+const items = new Array(3).fill().map((i, index) => `Option ${index + 1}`);
+
+describe('ContextualMenuDecorator Specs', () => {
+	test('should render component into FloatingLayer if open', () => {
+		const Root = FloatingLayerDecorator('div');
```
Comment (adrian-cocoara-lgp) [inline ContextualMenuDecorator/tests/ContextualMenuDecorator-specs.js:14]: How about moving this variable declaration above `describe`? This way you don't have to declare it at the beginning of each test.

**Review by adrian-cocoara-lgp — APPROVED** [review]
LGTM

**Pull request merged**